Include/Rover/RandomPick.hpp

```diff
@@ -38,6 +38,11 @@ namespace Rover {
   template<typename... GeneratorsFwd>
   RandomPick(GeneratorsFwd&&...) -> RandomPick<std::decay_t<GeneratorsFwd>...>;
 
+  //! GCC workaround.
```

Comment:
https://stackoverflow.com/questions/43430921/deduction-guides-and-variadic-class-templates-with-variadic-template-constructor